datahub-web-react/src/app/ingest/source/builder/RecipeBuilder.tsx

```diff
@@ -65,16 +73,57 @@ function RecipeBuilder(props: Props) {
         }
     }
 
+    let link: React.ReactNode;
+    if (type === 'looker') {
+        link = (
+            <a href={LOOKER_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub looker module
+            </a>
+        );
+    } else if (type === 'lookml') {
+        link = (
+            <a href={LOOKML_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub lookml module
+            </a>
+        );
+    }
+
     return (
         <div>
-            <ButtonsWrapper>
-                <StyledButton type="text" isSelected={isViewingForm} onClick={() => switchViews(true)}>
-                    <FormOutlined /> Form
-                </StyledButton>
-                <StyledButton type="text" isSelected={!isViewingForm} onClick={() => switchViews(false)}>
-                    <CodeOutlined /> YAML
-                </StyledButton>
-            </ButtonsWrapper>
+            {(type === 'looker' || type === 'lookml') && (
+                <Alert
+                    type="warning"
+                    banner
+                    message={
+                        <>
+                            <big>
+                                <i>
+                                    <b>You must acknowledge this message to proceed!</b>
+                                </i>
+                            </big>
+                            <br />
+                            <br />
+                            To get complete Looker metadata integration (including Looker views and lineage to the
+                            underlying warehouse tables), you must <b>also</b> use the {link}.
+                        </>
+                    }
+                />
+            )}
+            <Row style={{ marginBottom: '10px' }}>
+                <Col xl={8} lg={8} md={8} sm={24} xs={24}>
+                    <Title level={5}>{sourceConfigs?.displayName} Recipe</Title>
+                </Col>
+                <Col xl={16} lg={16} md={16} sm={24} xs={24}>
+                    <ButtonsWrapper>
+                        <StyledButton type="text" isSelected={isViewingForm} onClick={() => switchViews(true)}>
+                            <FormOutlined /> Form
+                        </StyledButton>
+                        <StyledButton type="text" isSelected={!isViewingForm} onClick={() => switchViews(false)}>
+                            <CodeOutlined /> YAML
+                        </StyledButton>
+                    </ButtonsWrapper>
+                </Col>
+            </Row>
```

Comment:
Nothing wrong with using `Row` and `Col` here - just curious why you decided on that instead of just wrapping `Title` and `ButtonsWrapper` in a parent `div` with `display: flex; justify-content: space-between; margin-bottom: 10px;`?

Reply:
Yes, this is another solution to implement this. I only used Row and Col at that time.